test/unit/events.js

```diff
@@ -66,4 +66,52 @@ describe('custom events', () => {
       done();
     }, 1);
   });
+
+  it('should handle cases correctly', done => {
+    let count = 0;
+    const Comp = reactify(document.registerElement('x-custom-event-4', {
+      prototype: Object.create(HTMLElement.prototype, {
+        trigger: {
+          value() {
+            this.dispatchEvent(new CustomEvent('testThis'));
+          },
+        },
+      }),
+    }), { React, ReactDOM });
+
+    const func = () => ++count;
+
+    // Using both ontest and onTest (case-sensitive) test case-sensitivity.
+    const comp = ReactDOM.render(<Comp ontestThis={func} onTestThis={func} />, window.fixture);
```

Comment:
I'm wondering if it'd be good to test the first character after on to make sure it's uppercase so that things like `once` don't get passed through as a `ce` event.